.rspec

```diff
@@ -0,0 +1,2 @@
+--color
```

Comment:
It's not much of an issue to have this included in the project repo, although you could have put this `.rspec` file in your home directory and it would work for your local machine (across all projects).